docs/doc.php

```diff
@@ -16,60 +16,33 @@ function endsWith($haystack, $needle) {
     exit();
   }
   if ($request_uri == "/doc/blog/index") {
-    header("Location: /doc/blog/Webots-2019-b-release");
+    header("Location: /doc/blog/Webots-2020-a-release");
```

Comment:
This link should be automatically updated (when updating the release version) or we should document to update it just before the release.

Reply:
Normally, it's done here: https://github.com/cyberbotics/webots/blob/master/src/bin/new_version.sh#L51
Do you think this doesn't work?

Reply:
I will check if it works.

Reply:
I checked and it works fine (also on the develop branch it is correctly set).
So then I don't know why it was wrongly set here.